### DISCUSSION TITLE
ImageQuery quality

### DISCUSSION THREAD

**Discussion by bdreyerr** [MacPaw/OpenAI#152] [General]
I've noticed when using the ImageQuery, there are only three parameters we can specify (prompt, n, size). However, I notice that on OpenAI's documentation for the DALLE model, you are able to specify model (DALLE3 vs. DALLE4). I'm wondering if there will ever plan to be support for this?

I'm asking because the quality of the images returned from the ImageQuery API is generally quite low compared to what you would expect. Not so much quality in terms of the actual resolution and pixel detail but the content of the image. It's as if the images returned were generated with less context than normal DALLE.

**Comment by kalafus** [comment]
supported as of 2024 Feb 15 [PR 169](https://github.com/MacPaw/OpenAI/pull/169).
